region/brinstar/red/Below Spazer.json

```diff
@@ -479,7 +479,8 @@
           "nodes": [1, 2],
           "mustStayPut": false
         }},
-        {"refill": ["Energy", "Super"]}
+        {"partialRefill": {"type": "Energy", "limit": 100}},
+        {"partialRefill": {"type": "Super", "limit": 5}}
```

Comment:
i wonder if there is a way we could model the energy as only if youre close to full on supers.